lib/response_bank.rb

```diff
@@ -25,8 +25,8 @@ def write_to_backing_cache_store(_env, key, payload, raw:)
       cache_store.write(key, payload, raw: raw)
     end
 
-    def read_from_backing_cache_store(_env, cache_key, cache_store: cache_store)
-      cache_store.read(cache_key)
+    def read_from_backing_cache_store(_env, cache_key, backing_cache_store: cache_store)
```

Comment:
Don't you need to change the caller as well?

Reply:
@rafaelfranca Apologies, didn't push. Done!